package.json

```diff
@@ -37,47 +37,50 @@
     "release:patch": "npm test && node ./scripts/release/release.js patch",
     "nightwatch": "nightwatch",
     "nightwatch:port-provided": "nightwatch -c tests/test.nightwatch.conf.js",
-    "test": "npm run nightwatch && npm run nightwatch:port-provided"
+    "test": "npm run compile && npm run nightwatch && npm run nightwatch:port-provided"
   },
   "dependencies": {
-    "autoprefixer": "^6.7.7",
-    "babel-loader": "^7.0.0",
-    "babel-polyfill": "^6.23.0",
     "chromedriver": "^2.31.0",
-    "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "file-loader": "^0.11.1",
     "glob": "^7.1.1",
-    "html-webpack-plugin": "^2.28.0",
-    "json-loader": "^0.5.4",
     "load-json-file": "^2.0.0",
-    "nightwatch": "^0.9.12",
-    "node-sass": "^4.5.2",
-    "postcss-loader": "^1.3.3",
-    "raw-loader": "^0.5.1",
-    "sass-loader": "^6.0.3",
-    "shelljs": "^0.7.6",
-    "style-loader": "^0.16.1",
-    "webpack": "^2.6.1",
-    "webpack-dev-server": "^2.4.5"
+    "shelljs": "^0.7.6"
```

Comment:
shelljs is just used in the release script. I think we can move this to be a devDep